.github/workflows/deploy-vault.yml

```diff
@@ -0,0 +1,62 @@
+name: deploy-vault.ci
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'src/pol-vaults/mainnet.json'
+
+permissions:
+  checks: write
+  contents: write
+
+env:
+  DEPLOYER_PRIVATE_KEY: ${{ secrets.DEPLOYER_PRIVATE_KEY }}
+
+jobs:
+  autofix:
+    name: Lints, handles new assets, and commits changes
```

Comment:
This is not autofixing and linting.

We should probably make this part of `lint` (and rename it)?